packages/react/src/create-strict-api/index.ts

```diff
@@ -146,7 +154,13 @@ interface CompiledAPI<TSchema extends CompiledSchema> {
       requiredProperties: TAllowedProperties;
       requiredPseudos: TAllowedPseudos;
     } = never
-  >(): Internal$XCSSProp<TAllowedProperties, TAllowedPseudos, TSchema, TRequiredProperties>;
+  >(): Internal$XCSSProp<
+    TAllowedProperties,
+    TAllowedPseudos,
+    TMediaQueries,
```

Comment:
Did we want to support any other at-rules, eg. `@supports`?  Can easily wait for an ask—easy to slot in later if necessary, nothing here really blocks it (aside from the one type).

Reply:
I'm not sure. For XCSS prop I don't think so. For strict API probably, but it would be an internal change.